bin/rdme.js

```diff
@@ -1,3 +1,5 @@
-#!/usr/bin/env node
+#!/usr/bin/env NODE_OPTIONS=--no-warnings node
```

Comment:
Does that other `process.env.NODE_NO_WARNINGS = 1;` solution work too? I'm not sure which one is cleaner.

Reply:
nope, doesn't work 😕